Content.Server/_Goobstation/Heretic/Ritual/CustomBehavior.Sacrifice.cs

```diff
@@ -213,11 +216,14 @@ private void TeleportRandomly(RitualData args, EntityUid uid) // start le telepo
             var randVector = randomSystem.NextVector2(maxrandomradius);
             newCoords = coords.Offset(randVector);
 
+            xformSystem.SetCoordinates(uid, newCoords); //move person teleported to check if they're under a tile (the tiles intersecting doesnt account for this so we need to do this) -space
```

Comment:
How did this look in your tests, was the person teleported around several times?

Reply:
from my tests it works instantly and doesn't show them being teleported around the map